src/software/thunderscope/constants.py

```diff
@@ -5,6 +5,9 @@
 SPEED_LINE_WIDTH = 2
 SPEED_SEGMENT_SCALE = 0.2
 
+ROBOT_RADIUS = 25
+TEAM_B = "b"
```

Comment:
I think the "b" here is just referring to the color blue for the center circle. We can probably either just hard code the color to "blue" (a bit more descriptive) or move this to the Colors class below

Reply:
done